test/end2end_test.go

```diff
@@ -1037,7 +1043,10 @@ func (s) TestDetailedConnectionCloseErrorPropagatesToRpcError(t *testing.T) {
 	// the RPC has been started on it.
 	<-rpcStartedOnServer
 	ss.S.Stop()
-	if _, err := stream.Recv(); err == nil || (!strings.Contains(err.Error(), possibleConnResetMsg) && !strings.Contains(err.Error(), possibleEOFMsg)) {
+	// The precise behavior of this test is subject to raceyness around the timing of when TCP packets
+	// are sent from client to server, and when we tell the server to stop, so we need to account for both
+	// of these possible error messages
```

Comment:
Same here: Please wrap all block comments to 80 columns.

Reply:
From one of my earlier PRs on the team (assuming you use JetBrains IDE's which are by far the best): [menghanl](https://github.com/menghanl) [on Apr 13, 2021](https://github.com/grpc/grpc-go/pull/4332#discussion_r612653948)
I like to wrap the comments (but only the comments) to 80.

And I use this: https://plugins.jetbrains.com/plugin/7234-wrap-to-column

Reply:
> assuming you use JetBrains IDE's which are by far the best

Subjectivity !!